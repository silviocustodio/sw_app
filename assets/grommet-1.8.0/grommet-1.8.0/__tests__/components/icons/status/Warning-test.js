// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP

import React from 'react';
import renderer from 'react-test-renderer';

import Warning from
  '../../../../src/js/components/icons/status/Warning';

describe('Warning', () => {
  it('has correct default options', () => {
    const component = renderer.create(
      <Warning />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('has correct className rendering', () => {
    const component = renderer.create(
      <Warning className='testing' />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
