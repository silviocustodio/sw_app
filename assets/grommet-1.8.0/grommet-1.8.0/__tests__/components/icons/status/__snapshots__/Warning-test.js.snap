// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Warning has correct className rendering 1`] = `
<svg
  aria-label="Warning"
  className="grommetux-status-icon grommetux-status-icon-warning testing"
  role="img"
  version="1.1"
  viewBox="0 0 24 24"
>
  <defs>
    <mask
      id="mask-warning-1"
    >
      <g
        className="grommetux-status-icon__detail"
      >
        <rect
          fill="#fff"
          height="24"
          width="24"
          x="0"
          y="0"
        />
        <g
          stroke="#000"
          strokeWidth="2"
          transform="translate(11.000000, 8.000000)"
        >
          <path
            d="M1,0 L1,6"
            fill="none"
            role="presentation"
          />
          <path
            d="M1,8 L1,10"
            fill="none"
            role="presentation"
          />
        </g>
      </g>
    </mask>
  </defs>
  <g
    className="grommetux-status-icon__base"
    mask="url(#mask-warning-1)"
  >
    <path
      d="M12,0 L0,22 L24,22 L12,0 L12,0 Z"
      role="presentation"
      stroke="none"
    />
  </g>
</svg>
`;

exports[`Warning has correct default options 1`] = `
<svg
  aria-label="Warning"
  className="grommetux-status-icon grommetux-status-icon-warning"
  role="img"
  version="1.1"
  viewBox="0 0 24 24"
>
  <defs>
    <mask
      id="mask-warning-0"
    >
      <g
        className="grommetux-status-icon__detail"
      >
        <rect
          fill="#fff"
          height="24"
          width="24"
          x="0"
          y="0"
        />
        <g
          stroke="#000"
          strokeWidth="2"
          transform="translate(11.000000, 8.000000)"
        >
          <path
            d="M1,0 L1,6"
            fill="none"
            role="presentation"
          />
          <path
            d="M1,8 L1,10"
            fill="none"
            role="presentation"
          />
        </g>
      </g>
    </mask>
  </defs>
  <g
    className="grommetux-status-icon__base"
    mask="url(#mask-warning-0)"
  >
    <path
      d="M12,0 L0,22 L24,22 L12,0 L12,0 Z"
      role="presentation"
      stroke="none"
    />
  </g>
</svg>
`;
